images/cdi/k8s-rdma-shared-dev-plugin-cdi-ds.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: rdma-shared-dp-ds
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      name: rdma-shared-dp-ds
+  template:
+    metadata:
+      labels:
+        name: rdma-shared-dp-ds
+    spec:
+      hostNetwork: true
+      priorityClassName: system-node-critical
+      containers:
+        - image: ghcr.io/mellanox/k8s-rdma-shared-dev-plugin:latest-amd64
```

Comment:
This line is dependent on another opened PR: https://github.com/Mellanox/k8s-rdma-shared-dev-plugin/pull/75